admin/client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -134,27 +134,31 @@ class CampaignAdminList extends SilverStripeComponent {
       );
     });
 
-    // Get preview details
-    const classNames = itemLinks
-      ? 'cms-content__split cms-content__split--left-sm'
-      : 'cms-content__split cms-content__split--none';
+    // Set body
+    const body = accordionBlocks.length
+      ? (<Accordion>{accordionBlocks}</Accordion>)
+      : (
+        <div className="alert alert-warning" role="alert">
+          <strong>This campaign is empty.</strong> You can add pages by selecting{' '}
+          <em>Add to campaign</em> from within the <em>More Options</em> popup on{' '}
+          the edit page screen.
```

Comment:
I would be ok with "Edit this page" or "Edit this file", which then has both the section name and makes sense to a user but I don't really think "the Pages section" is meaningful enough, ie it doesn't say take me to the page with the pages section.

FYI this is just a temporary button for alpha1, well there are plans to make editing availalbe within campaigns.